crates/cairo-lint-core/tests/test_files/bitwise_for_parity_check/bitwise_for_parity_check

```diff
@@ -0,0 +1,25 @@
+//! > bitwise for parity check
```

Comment:
can you add a few more tests with more complex expressions plz and also prefix variables with `_` so we don't have the annoying unused variable diagnostic